backend/lambdas/generators/json_report/json_report/results/sbom.py

```diff
@@ -0,0 +1,22 @@
+from artemisdb.artemisdb.consts import PluginType
+from artemisdb.artemisdb.models import Scan
+from json_report.results.results import PLUGIN_RESULTS, PluginErrors
+
+
+def get_sbom(scan: Scan) -> PLUGIN_RESULTS:
+    """
+    Unify the output of sbom plugins. We deliberately omit the `details` property in the database
+    (since SBOM results are stored in s3), so this returns no scan results or summary
+    NOTE: unit tests are located at api/tests/test_generate_report.py
+    Inspect tests for expected output format.
+    :param scan: django object of Artemis repo scan
+    :return: dictionary of None for findings, list of errors, boolean success, and None for summary
+    """
+
+    errors = PluginErrors()
+
+    plugin = _empty = object()
```

Comment:
Can this be omitted since this generator does not have the `if plugin is _empty` block that the other generators have?

Reply:
Good catch